waf_tools/corrade.py

```diff
@@ -152,12 +152,10 @@ def find_in_string(data, text):
 
                 corrade_component_libpaths[component] = corrade_component_libpaths[component] + [lib_dir]
                 corrade_component_libs[component].append(lib)
-                if component == 'PluginManager':
-                    # PluginManager needs the libdl.so library
+                if component == 'PluginManager' or component == 'Utility':
+                    # PluginManager/Utility need the libdl.so library
```

Comment:
minor: because PluginManager depends on Utility, only Utility need the `dl` lib, PluginManager will get it transitively :)